test/MC/RISCV/rvc.s

```diff
@@ -0,0 +1,17 @@
+# RUN: llvm-mc %s -triple=riscv32imac -show-encoding \
+# RUN:     | FileCheck -check-prefixes=CHECK,CHECK-INST %s
+# RUN: llvm-mc -filetype=obj -triple riscv32imac < %s \
+# RUN:     | llvm-objdump -d - | FileCheck -check-prefix=CHECK-INST %s
+
```

Comment:
This test should be a little more exhaustive, e.g. checking the limits of immediate ranges

Reply:
Add rvc-invalid.s MC testcase to test the limits of immediate ranges.